sdk/communication/azure-communication-rooms/ci.yml

```diff
@@ -28,7 +28,7 @@ pr:
 extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
-    ServiceDirectory: communication
+    ServiceDirectory: communication/azure-communication-rooms
```

Comment:
I worry this might break some of the usages of ServiceDirectory in ci.yml or release templates. So this is something we should be on the lookout for and ensure we don't break.

Reply:
@weshaggard being on the lookout is something I'd do anyways (yup, no Type-A here). I don't believe this is going to be a problem. The common steps like daily-dev-build-variable, verify-readme, verify-changelog, already had to work in this case, otherwise Go would be broken. I was diligent in my testing. The only case I couldn't test was an official release of a pipeline whose ServiceDirectory was `<ServiceDirectory>/<LibraryDirectory>`, since I'm not creating another template library/pipeline for this case. With that being said, I was able to go through the nightly release path for that scenario and the couple of steps that don't run in the nightly release, which do run in the official release. Those additional steps, which only consisted of the scan_for_unreleased_dependencies.ps1 and verify_release_set.ps1, I was able to test by hand. The only thing that ServiceDirectory does, in both cases, is that it narrows down what needs to be scanned in the case where the ci.yml is in the library directory.